galvan-support/src/main/java/org/terracotta/testing/config/StartupBuilder.java

```diff
@@ -1,32 +0,0 @@
-/*
```

Comment:
Keep the interface. It is less code change

Reply:
I see two possibilities:
1. we have an interface and tc-core and tc-platform implement the methods with no commonality between them. This is ideal when there's no common code.
2. We have the common code in the abstract class, and only the different part in the subclasses. Everything apart from the `build()` method is the same, so I went for an abstract class.

Reply:
It’s always nice to program against interfaces event if you have an abstract for convenience and code reuse.  List -> AbstractList is a good example in the jerk collections library.

Reply:
Kept the interface as is, and added getters in `DefaultStartupCommandBuilder` so that subclasses can use them.